From b610e4667dcd201915e8c1c9889d3f5726c8cf4b Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Mon, 16 Jul 2018 11:39:15 +0300
Subject: [PATCH 0665/1051] dts: a8040-db: include 8040-db.dtsi instead of
 8040.dtsi

We should use the following structure in the device-tree:
        armada-8040.dtsi - SoC common definitions
        armada-8040-db.dtsi - Board common definitions.
        armada-8040-db-X.dts - Specific board definitions

Currently, in armada-8040-db.dts, we include armada-8040.dtsi
directly, skipping armada-8040-db.dtsi which creates duplications
in the device-tree.

This patch fixes this issue, buy replacing the include in
armada-8040-db.dts.

The following definitions are added with this change:
- spi-flash@0
- several sdhci@780000 definitions.
- cp0_vccq_sd0
- several NAND definitions

Cleanup of the duplication will be done in a separate patch.

Change-Id: I0fefbbea9f91186ba5265e755f1e2a9565deb945
Signed-off-by: Igal Liberman <igall@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-8040-db.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db.dts b/arch/arm64/boot/dts/marvell/armada-8040-db.dts
index fddafd679812..e47405d74ac3 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db.dts
@@ -6,7 +6,7 @@
  */
 
 #include <dt-bindings/gpio/gpio.h>
-#include "armada-8040.dtsi"
+#include "armada-8040-db.dtsi"
 
 / {
 	model = "Marvell Armada 8040 DB board";
-- 
2.17.1

